install.hs

```diff
@@ -172,7 +172,7 @@ helpMessage = do
   let out = liftIO . putStrLn
   out ""
   out "Usage:"
-  out "    stack Shakefile.hs <target>"
+  out "    stack install.hs <target>"
```

Comment:
@fendor I wonder if hardcoded filename should be replaced with something like `argv[0]`. I doubt this file would be renamed again anytime soon, but one less place to check is a good thing.

Reply:
I agree, that would be better.
In the next 2 weeks I'd like to add various improvements to it.